test/integration/ItemsTest.php

```diff
@@ -184,92 +194,77 @@ public function testIsItemClass()
         $this->assertTrue($this->itemsService->isItemClass($clazz->reveal()));
     }
 
-    // @todo fix unexpected method call
     public function testGetModelRuntime()
     {
-        $item = $this->prophesize('core_kernel_classes_Resource');
-        $itemModel = $this->prophesize('core_kernel_classes_Resource');
-        $itemModel->getOnePropertyValue(new core_kernel_classes_Property(taoItems_models_classes_itemModel::CLASS_URI_RUNTIME))
-            ->willReturn('returnValue');
-        $item->getOnePropertyValue(new core_kernel_classes_Property(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL))
-            ->willReturn($itemModel->reveal());
-        
-        $this->assertEquals('returnValue', $this->itemsService->getModelRuntime($item->reveal()));
+        $item = $this->ontologyMock->getResource(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL);
```

Comment:
It is confusing to have same uri for two different resources, also `new core_kernel_classes_resource` can be replaced `$this->ontologyMock->getResource`